lib/rubocop-rspec.rb

```diff
@@ -37,16 +37,36 @@
 # We have to register our autocorrect incompatibilities in RuboCop's cops
 # as well so we do not hit infinite loops
 
+# rubocop:disable Style/Documentation
 module RuboCop
   module Cop
     module Layout
-      class ExtraSpacing # rubocop:disable Style/Documentation
+      class ExtraSpacing
         def self.autocorrect_incompatible_with
           [RSpec::AlignLeftLetBrace, RSpec::AlignRightLetBrace]
         end
       end
     end
+
+    module Style
+      class TrailingCommaInArguments
+        def self.autocorrect_incompatible_with_ext
+          TrailingCommaInArguments.autocorrect_incompatible_with_org
+            .push(RSpec::Capybara::CurrentPathExpectation)
+        end
+
+        def autocorrect_incompatible_with_extend(*args)
+          TrailingCommaInArguments.autocorrect_incompatible_with_extend(*args)
+        end
+
+        class << self
+          alias autocorrect_incompatible_with_org autocorrect_incompatible_with
+          alias autocorrect_incompatible_with autocorrect_incompatible_with_ext
+        end
```

Comment:
Instead of an alias-method chain, we could do module-prepending which I think is cleaner:
```rb
RuboCop::Cop::Layout::ExtraSpacing.singleton_class.prepend(Module.new do
  def autocorrect_incompatible_with
    [RSpec::AlignLeftLetBrace, RSpec::AlignRightLetBrace]
  end
end)

RuboCop::Cop::Style::TrailingCommaInArguments.singleton_class.prepend(Module.new do
  def autocorrect_incompatible_with
    [RSpec::Capybara::CurrentPathExpectation]
  end
end)
```

Of course this is very much repetitive, so perhaps later we should consider adding a class method to `Cop` that can add all this code with a single call. But definitely not for this PR though.

Reply:
IMO it should be `append` and concating to the `super`, otherwise it will still be just replaced by the original code

Reply:
🤦🏼 Of course, I’m not sure how I managed to write down only half of my thoughts before commenting. Thanks for catching that @Darhazer. What I meant was something along the lines of
```rb
RuboCop::Cop::Layout::ExtraSpacing.singleton_class.prepend(Module.new do
  def autocorrect_incompatible_with
    super.push(RSpec::AlignLeftLetBrace).push(RSpec::AlignRightLetBrace)
  end
end)

RuboCop::Cop::Style::TrailingCommaInArguments.singleton_class.prepend(Module.new do
  def autocorrect_incompatible_with
    super.push(RSpec::Capybara::CurrentPathExpectation)
  end
end)

Reply:
Thanks. I'll use module-prepending, as it certainly feels cleaner that way.